playground/multi-camera.ts

```diff
@@ -0,0 +1,121 @@
+/**
+ * @title Multi Camera
+ * @category Camera
+ */
+
+import {
+  CameraClearFlags,
+  DirectLight,
+  AssetType,
+  Entity,
+  Camera,
+  WebGLEngine,
+  Vector3,
+  MeshRenderer,
+  BlinnPhongMaterial,
+  Color,
+  BackgroundMode,
+  SkyBoxMaterial,
+  PrimitiveMesh,
+  Layer,
+  Script,
+  TextureCubeMap
+} from "oasis-engine";
+import {OrbitControl} from "@oasis-engine/controls";
+import {SpineAnimation} from "@oasis-engine/spine";
+
+/**
+ * Script for rotate.
+ */
+class RotateScript extends Script {
+  /**
+   * @override
+   * The main loop, called frame by frame.
+   * @param _deltaTime - The deltaTime when the script update.
+   */
+  onUpdate(_deltaTime: number): void {
+    this.entity.transform.rotate(0.0, 0.6, 0);
+  }
+}
+
+const engine = new WebGLEngine("canvas");
+engine.canvas.resizeByClientSize();
+const scene = engine.sceneManager.activeScene;
+const {background} = scene;
+const rootEntity = engine.sceneManager.activeScene.createRootEntity();
+
+// init full screen camera
+const cameraEntity = rootEntity.createChild("fullscreen-camera");
+const camera = cameraEntity.addComponent(Camera);
+camera.cullingMask = Layer.Layer0;
+camera.clearFlags = CameraClearFlags.Depth;
```

Comment:
This is the scene camera, I think the first camera should clear `Color`( enum `DepthColor`, default value), or the scene will be left over dirty color when camera move next frame